SPECS/hyper-v/hyper-v.spec

```diff
@@ -30,9 +30,9 @@ install -vdm 755 %{buildroot}/%{_sbindir}
 install -m 755  %{SOURCE5}  %{buildroot}/%{_sbindir}/lsvmbus
 install -vdm 755 %{buildroot}/%{_bindir}
 install -m 755  %{SOURCE1}  %{buildroot}/%{_bindir}/bondvf.sh
-install -m 755  %{SOURCE2}  %{buildroot}/%{_bindir}/hv_get_dns_info.sh
-install -m 755  %{SOURCE3}  %{buildroot}/%{_bindir}/hv_get_dhcp_info.sh
-install -m 755  %{SOURCE4}  %{buildroot}/%{_bindir}/hv_set_ifconfig.sh
+install -m 755  %{SOURCE2}  %{buildroot}/%{_bindir}/hv_get_dns_info
+install -m 755  %{SOURCE3}  %{buildroot}/%{_bindir}/hv_get_dhcp_info
+install -m 755  %{SOURCE4}  %{buildroot}/%{_bindir}/hv_set_ifconfig
```

Comment:
Thank @cocowalla for reporting the issue and patch to solve this issue.

hv_kvp_daemon.c looks for these scripts at KVP_SCRIPTS_PATH i.e. "/usr/libexec/hypervkvpd/". If you have the setup could you please verify this.

Reply:
Hmm, then is `/usr/libexec/hypervkvpd` a directory symlink pointing to `/usr/bin`?

In my Photon 3.0 Rev2 kickstart file (without this change), I have this as a (working!) workaround:

```json
  "postinstall": [
    "#!/bin/sh",
    "tdnf install -y sudo hyper-v",
    "ln -s /usr/bin/hv_get_dhcp_info.sh /usr/bin/hv_get_dhcp_info",
    "ln -s /usr/bin/hv_get_dns_info.sh /usr/bin/hv_get_dns_info",
    "ln -s /usr/bin/hv_set_ifconfig.sh /usr/bin/hv_set_ifconfig",
    "systemctl enable hv_kvp_daemon",
    "systemctl enable hv_vss_daemon"
  ],
```

I don't have a Photon VM available right now, but I'll try to check on the `KVP_SCRIPTS_PATH` thing later.